ui/raidboss/data/03-hw/dungeon/fractal_continuum.txt

```diff
@@ -0,0 +1,156 @@
+hideall "--reset--"
+hideall "--sync--"
+hideall "--start--"
+
+0 "--reset--" sync /:Exhibit level III is no longer sealed/ window 10000 jump 0
+0 "--reset--" sync /:The high-level incubation bay is no longer sealed/ window 10000 jump 0
+0 "--reset--" sync /:The reality augmentation bay is no longer sealed/ window 10000 jump 0
+
+
+###Phantom Ray
+#-ii F7D
+
+0 "Start"
+0.0 "--start--" sync /Exhibit level III will be sealed off/ window 0,1
+10.3 "Rapid Sever" sync /:Phantom Ray:F7A:/
+16.6 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/
+21.4 "Double Sever 1" sync /:Phantom Ray:F7B:/
+25.6 "Double Sever 2" sync /:Phantom Ray:F7C:/
+27.4 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/
+
+41.5 "Rapid Sever" sync /:Phantom Ray:F7A:/
+47.7 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/
+52.5 "Double Sever 1" sync /:Phantom Ray:F7B:/
+56.5 "Double Sever 2" sync /:Phantom Ray:F7C:/
+58.3 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/ window 8,10 jump 27.4
+
+72.4 "Rapid Sever" sync /:Phantom Ray:F7A:/ jump 41.5
+78.6 "Atmospheric Displacement"
+83.3 "Double Sever 1"
+87.4 "Double Sever 2"
+89.2 "Atmospheric Displacement"
+#Phase 2 at < 75% HP
+
+97.7 "--sync--" #landing for phase 2 loop
```

Comment:
I think in general, I try to have abilities loop back to themselves rather than to an empty spot in time.  Can you just have the single jump back to 100? Instead of one jump to 97.7 and another jump to 100.0?

I guess I don't quite understand the double jumps everywhere.

Reply:
Same as the earlier jump, in terms of why it's doubled.

This is also the start of phase 2, so I think I was trying to make it start on an even 100? That makes the exact phase length easy to see. The sync to an empty line allows that to happen. I did the same thing for Curator but not for Minotaur, so I'm unsure now what I was thinking. Regardless, it should be easy to change to whatever you prefer.

Reply:
Actually, this is a question. It's currently set up as:

```
139.0 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/
146.1 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/ jump 97.7

148.4 "Damage Up" sync /:Phantom Ray:F7F:/ jump 100.0
153.2 "Double Sever 1"
```

Stylistically, I suppose it's better to jump from the start of the lookahead then? That is, leave the jump at 148.4 and remove the one at 146.1? That would remove the need for a landing point before the loop.

Reply:
I usually jump from the start of the lookahead, just because that's the usually the first skill you can pick where you can jump back to that same skill earlier in the timeline.  If that doesn't have an id or is something you don't want to sync against for various reasons, picking some other skill in the lookahead is fine too.

I think the important part for me is to jump from one skill to itself earlier.

Reply:
Great. Thank you.